Server/src/controllers/EntriesController.js

```diff
@@ -2,6 +2,7 @@ import moment from 'moment';
 import database from '../config/databaseConnection';
 import find from '../models/queries/find.json';
 import entriesDb from '../models/dummy-db/Entries.json';
```

Comment:
Why is this file  `json`?

Reply:
entriesDb is a file from my dummy database I will flush out ASAP,